### DISCUSSION TITLE
jc + VisiData make `jcv`

### DISCUSSION THREAD

**Discussion by frosencrantz** [kellyjonbrazil/jc#301] [Show and tell]
Wanted to mention that `jc` works great with @saulpw's  VisiData http://github.com/saulpw/visidata:

I have this simple script for viewing JC output in VisiData:
```
#!/usr/bin/env bash
jc "$@" | vd -n -f json
```

Here is an example using the `--about` flag to see the output of the new `--proc` flags can show:

https://asciinema.org/a/jYMMexlcU76ICMfmZIuXn284p

It loads `jc --about` into VisiData, looks at the all the parsers.  Find the /proc ones that are not specific to a pid.  Pulls on the /proc/filename.  Runs `jc` with the flag and the filename as input, and the parses the output as JSON.  You can now explore the results in VisiData.

**Comment by kellyjonbrazil** [comment]
Very cool! I was about to tweet this out but I noticed the video was marked private, so holding off.

**Reply by frosencrantz** [reply]
You are welcome to share on Twitter.  I usually use Asciinema for the bug reports, and don't spend much time to make sure they are public.  I've made this one public, and gave it a name, with a link back to this thread.
